### ISSUE TITLE
Cursor visible

### ISSUE THREAD

**Issue by wighawag** [ccontavalli/grub-shusher#12]
I used grub-susher succesfuly to remov ethe messages but it seems it still print some newline and the cursor is visbile.

see video:

[cursor.webm](https://github.com/user-attachments/assets/afd86e82-58af-4a5e-af24-c7f0cb418886)

Amyone else seen that ?

**Comment by stradicat** [comment]
Yes, I guess it's a GRUB quirk that can't be mitigated by the script (yet).

I may fork it to check it anything can be done.
